Source/Core/DolphinWX/ControllerConfigDiag.cpp

```diff
@@ -227,7 +227,7 @@ wxSizer* ControllerConfigDiag::CreateWiimoteConfigSizer()
 
   auto* const box = new wxStaticBoxSizer(wxVERTICAL, this, _("Wii Remotes"));
 
-  m_passthrough_bt_radio = new wxRadioButton(this, wxID_ANY, _("Passthrough a Bluetooth adapter"),
+  m_passthrough_bt_radio = new wxRadioButton(this, wxID_ANY, _("Passthrough a Bluetooth Adapter"),
```

Comment:
I don't know, it just looks odd to me. Feel free to ignore this though, it's just my opinion.